stdlib/2/typing.pyi

```diff
@@ -62,46 +63,58 @@ _KT_co = TypeVar('_KT_co', covariant=True)  # Key type covariant containers.
 _VT_co = TypeVar('_VT_co', covariant=True)  # Value type covariant containers.
 _T_contra = TypeVar('_T_contra', contravariant=True)  # Ditto contravariant.
 
-class SupportsInt(metaclass=ABCMeta):
+def runtime(cls: _T) -> _T: ...
```

Comment:
``Type[Protocol]`` will not work, since it does not appear explicitly in MRO, but ``Type[object]`` looks like a good idea.